src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/otel-adot-log-ingestion-serverless.mdx

```diff
@@ -0,0 +1,191 @@
+
+
+## Pre-requisites
+
+* Make sure you have a <InlinePopover type="licenseKey" />.
+
+
+
+## Procedure
+
+The following setup shows one approach for configuring environment variables to your AWS Lambda application. You can also configure them from the [Functions](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) page.
+
+
+<Steps>
+
+<Step>
+
+Open the [AWS Serverless Application Repository](https://serverlessrepo.aws.amazon.com/applications) in your browser, and complete the following procedure.
+
+
+
+1. Under **Applications**, type `newrelic` in the search bar, and click the <DNT>**Show apps that create custom IAM roles or resource policies**</DNT> check-box to find `newrelic-aws-otel-log-ingestion`. 
+
+
+
+
+2. Open the `newrelic-aws-otel-log-ingestion` details and click <DNT>**Deploy**</DNT>.
+
+
+3. In the function's <DNT>**Configure**</DNT> menu, go to <DNT>**Environment Variables**</DNT> and configure log forwarding using the following environment variables: 
+
+
+<table>
+      <thead>
+        <tr>
+          <th style={{ width: "200px" }}>
+            Key
+          </th>
+    
+          <th>
+            Description
+          </th>
+        </tr>
+      </thead>
+    
+      <tbody>
+
+        <tr>
+          <td>
+            `OTEL_LOGGING_ENABLED`
+          </td>
+    
+          <td>
+            A boolean to determine if you want to output debug messages in the CloudWatch console. **Optional**.
+    
+            To turn on debug logs, set this to `true`. Default is `false`.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `DEBUG_LOGGING_ENABLED`
+          </td>
+    
+          <td>
+            A boolean to determine if you want to output debug messages in the CloudWatch console. **Optional**.
+    
+            To turn on debug logs, set this to `true`. Default is `false`.
+          </td>
+        </tr>
+    
+        <tr>
+          <td>
+            `LICENSE_KEY`
+          </td>
+    
+          <td>
+            <InlinePopover type="licenseKey" /> is used for sending data to New Relic. **Required.**
+          </td>
+        </tr>
+    
+        <tr>
+          <td>
+            `LOGGING_ENABLED`
+          </td>
+    
+          <td>
+            Set this flag to `false`.
+          </td>
+        </tr>
+    
+        <tr>
+          <td>
+            `NR_TAGS`
+          </td>
+    
+          <td>
+            Specify tags to be added to all log events. **Optional**.
+    
+            Each tag is composed of a colon-delimited key and value. Multiple key-value pairs are semicolon-delimited; for example, `env:prod;team:myTeam`.
+          </td>
+        </tr>
+    
+        <tr>
+          <td>
+            `NR_LAMBDA_LOG_GROUP_PREFIX`
+          </td>
+    
+          <td>
+            If you've configured your Lambda log group to use a different path than `/aws/lambda`, you can specify it with this variable. **Optional**.
```

Comment:
There are ways that we can simplify language to make it easier to read. For example:

"If you've configured your Lambda log group *with* a different path than `/aws/lambda`, you can *use* this variable."

This is a useful resource :) https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/